Translate "Avkrysningsboks" into "Choice box" (#568)
Altinn/altinn-studio-docs: content/app/design/guidelines/components/checkbox/_index.en.md

---

> title: Check box

Reviewer: This component is named `Checkboxes` in code, so the heading should at least be consistent with that.
```suggestion
title: Checkboxes
```

> description: Check boxes are used in cases where the user can mark multiple alternatives in a list of options.

Reviewer: ```suggestion
description: Check boxes are used in cases where the user can select multiple alternatives in a list of options.
```

weight: 50
---

> If the user only has one option, use the [radial buttons](../radiobutton/) instead. 

Reviewer: ```suggestion
If the user should only be able to pick one option, use [radio buttons](../radiobutton/) instead.
```
